src/tests/ftest/osa/online_drain.py

```diff
@@ -31,13 +32,14 @@ def setUp(self):
         self.dmg_command.exit_status_exception = True
         self.pool = None
 
-    def run_online_drain_test(self, num_pool, oclass=None, app_name="ior"):
+    def run_online_drain_test(self, num_pool, oclass=None, app_name="ior", multiple_ranks=None):
         """Run the Online drain without data.
 
         Args:
              num_pool (int) : total pools to create for testing purposes.
              oclass (str) : Object class type (RP_2G1, etc)
              app_name (str) : application to run on parallel (ior or mdtest). Defaults to ior.
+             multiple_ranks (list): List of multiple ranks online drain testing (Default: None)
```

Comment:
This would be better implemented as a number of ranks to drain
```suggestion
             num_ranks (int): Number of ranks to drain. Defaults to 1.
```

Reply:
done,